### PR TITLE
added image to list view of services

### PR DESCRIPTION
**Pull request by dasari-ananya**

### CODE REVIEW

File: src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/ListView/index.js [hunk 1]
```diff
@@ -1,52 +1,66 @@
...
 import CardContent from "@material-ui/core/CardContent";
 import CardActions from "@material-ui/core/CardActions";
 import Typography from "@material-ui/core/Typography";
 import MoreVertIcon from "@material-ui/icons/MoreVert";
 import IconButton from "@material-ui/core/IconButton";
-
 import { useStyles } from "./styles";
 
-const ListView = props => {
+function ListView(props) {
```
Comment (Vivek205) [inline src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/ListView/index.js]: let it be es6 function.
const ListView = props => {

File: src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/ListView/index.js [hunk 2]
```diff
@@ -1,52 +1,66 @@
...
-            </CardActions>
+                    <CardContent className={classes.cardContent}>
+                        <div className={classes.ratingSection}>
+                            <span className={classes.cardSubheader}>{props.cardSubheader}</span>
+                            <StarRatingComponent
+                                name="rate1"
+                                starCount={5}
+                                value={3}
+                                starColor={"#FFC200"}
+                                emptyStarColor={"rgba(161,163,168,0.35)"}
```
Comment (Vivek205) [inline src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/ListView/index.js]: move this color to theme.js and let sujith know about this color.

File: src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/index.js [hunk 3]
```diff
@@ -4,42 +4,49 @@ import { withStyles } from "@material-ui/styles";
...
 import { useStyles } from "./styles";
 
 class CardGroup extends Component {
     render() {
         const { classes } = this.props;
         return (
             <div className={classes.cardCollection}>
-                <StyledCard
+                <ListView
+                    cardMedia={CardImg}
```
Comment (Vivek205) [inline src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/index.js]: In ***future***, while creating the repeated contents of same type.. e.g. ListView is repeating here..  Make an array of objects and map it.

File: src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/index.js [hunk 4]
```diff
@@ -4,42 +4,49 @@ import { withStyles } from "@material-ui/styles";
 import ListView from "./ListView";
 import StyledCard from "./StyledCard";
 
+import CardImg from "../../../../../assets/images/dummy-card.png";
+
 import { useStyles } from "./styles";
 
 class CardGroup extends Component {
```
Comment (Vivek205) [inline src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/index.js]: It doesn't have any state, so better keep it functional component.

After converting it into functional components, check the styling once. coz useStyles have to be a hook, then.

File: src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/ListView/index.js [hunk 5]
```diff
@@ -1,52 +1,66 @@
...
 import Card from "@material-ui/core/Card";
+import CardMedia from "@material-ui/core/CardMedia";
 import CardHeader from "@material-ui/core/CardHeader";
 import CardContent from "@material-ui/core/CardContent";
 import CardActions from "@material-ui/core/CardActions";
 import Typography from "@material-ui/core/Typography";
 import MoreVertIcon from "@material-ui/icons/MoreVert";
 import IconButton from "@material-ui/core/IconButton";
-
 import { useStyles } from "./styles";
```
Comment (Vivek205) [inline src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/ListView/index.js]: keep the external packages imports together and userfiles imports together.

File: src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/index.js (hunk 3 above)
Comment (dasari-ananya) [reply src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/index.js]: i have just created that for a temp purpose, cause the data is going to be coming from the api

File: src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/index.js [hunk 6]
```diff
@@ -4,52 +4,57 @@ import { withStyles } from "@material-ui/styles";
 import ListView from "./ListView";
 import StyledCard from "./StyledCard";
 
+import CardImg from "../../../../../assets/images/dummy-card.png";
+
 import { useStyles } from "./styles";
```
Comment (Vivek205) [inline src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/index.js:7]: Will club all the userImports together. So need no new lines from line4 to line9.

**Review by Vivek205 — APPROVED** [review]

**Pull request merged**